Test/linters/constructorMatch.dfy.expect

```diff
@@ -0,0 +1,10 @@
+/Users/tobiaja/Documents/dafny/Test/linters/constructorMatch.dfy(11,12): Warning: this branch is redundant 
+/Users/tobiaja/Documents/dafny/Test/linters/constructorMatch.dfy(17,12): Warning: this branch is redundant 
+/Users/tobiaja/Documents/dafny/Test/linters/constructorMatch.dfy(25,16): Warning: this branch is redundant 
+/Users/tobiaja/Documents/dafny/Test/linters/constructorMatch.dfy(9,17): Warning: Variable name 'ShadesOfGray' on line 10 starts with a capital letter
```

Comment:
Note that the line number is mentioned twice, first in `(9,17)` and then in `on line 10`